meteor/client/ui/RundownView.tsx

```diff
@@ -1199,7 +1200,7 @@ export const RundownView = translateWithTracker<IProps, IState, ITrackedProps>((
 		playlist,
 		studio: studio,
 		showStyleBase,
-		showStyleVariant: rundowns.length > 0 ? ShowStyleVariants.findOne(rundowns[0].showStyleVariantId) : undefined,
+		showStyleVariant: rundowns.length > 0 ? ShowStyleVariants.findOne(showStyleVariantId) : undefined,
```

Comment:
The check for rundowns.length > 0 is redundant now, since you now uses the variant id you fetched further above